src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1203,4 +1205,29 @@
                         StablePackages="@(StablePackage)"
                         ModuleToPackages="@(ModuleToPackage)" />
   </Target>
+
+  <Target Name="CreateLayout"
```

Comment:
I'm fine with this for now, as you say we can pivot as needed. One nice thing about doing it as part of the individual build is that it will always update if you just rebuild the one library package project. However it might actually be interesting to it from the library build itself.